library/src/main/java/com/chuckerteam/chucker/internal/data/model/DialogData.kt

```diff
@@ -0,0 +1,8 @@
+package com.chuckerteam.chucker.internal.data.model
+
+internal data class DialogData(
```

Comment:
I don't see much value in adding this class.

Reply:
if I remove this data class [showDialog](https://github.com/ChuckerTeam/chucker/pull/305/files/5e1275ab62a02055be52d8a50156637cc27dedf8#diff-98f2123ddac8875c70a7ec16e42d5a1dR8) will accepting 6 parameters.
1. title
2. message
3. positive button text
4. negative button text
5. onPositiveClick
6. onNegativeClick

which causes detekt to fail the build with ` LongParameterList` message. however, I can remove 4 and 6 since its usage in this project is less, but in order to keep the method more generic, I have added those params.

I was not able to think of any other solution to avoid these parameters, But I am open to any suggestions you guys might have here.

Reply:
Ok, let's leave it.